apps/condo/bin/sbbol/migrateTokensToNewKeyPattern.js

```diff
@@ -0,0 +1,214 @@
+        const refreshTokenValue = await this.#getValue(`user:${userId}:refreshToken`)
+
+        return {
+            [organization]: organizationValue,
+            [clientSecretScopedKey]: clientSecretValue,
+            [accessTokenScopedKey]: accessTokenValue,
+            [refreshTokenScopedKey]: refreshTokenValue,
+        }
+    }
+
+    #getValue (key) {
+        const scopedKey = this.#scopedKey(key)
+        return this.keyStorage.get(scopedKey)
+    }
+
+    async #setValue (key, value, options = {}) {
+        const { ttl, expiresAt } = options
+        const commands = this.keyStorage.multi()
+        const scopedKey = this.#scopedKey(key)
+        commands.set(scopedKey, value)
+        if (ttl) {
+            commands.expire(scopedKey, ttl)
+        }
+        if (expiresAt) {
+            commands.expireat(scopedKey, expiresAt)
+        }
+        return commands.exec()
+            .then(() => {
+                console.log({ msg: `Set ${scopedKey}`, value })
+            }).catch(() => {
+                console.error({ msg: `Error set ${scopedKey}`, value })
+            })
+    }
+
+    async #isExpired (key) {
+        // NOTE: `TTL` Redis command returns -2 if the key does not exist, -1 if the key exists but has no associated expire
+        return (await this.keyStorage.ttl(this.#scopedKey(key))) <= 0
+    }
+
+    #scopedKey (key) {
+        return [SBBOL_REDIS_KEY_PREFIX, this.apiName, this.clientId, key].join(':')
+    }
+}
+
+
+async function getAccessTokenForUserOld (userId) {
+    const [clientId] = process.argv.slice(3)
+    const sbbolSecretStorage = new SbbolSecretStorage('auth', clientId)
+    if (await sbbolSecretStorage.isRefreshTokenExpired(userId)) {
+        const instructionsMessage = 'Please, login through SBBOL for this organization, so its accessToken and refreshToken will be obtained and saved in TokenSet table for further renewals'
+        throw new Error(`refreshToken is expired for clientId = ${sbbolSecretStorage.clientId}. ${instructionsMessage}`)
+    }
+
+    if (await sbbolSecretStorage.isAccessTokenExpired(userId)) {
+        const clientSecret = await sbbolSecretStorage.getClientSecret()
+        const currentRefreshToken = await sbbolSecretStorage.getRefreshToken(userId)
+        const oauth2 = new SbbolOauth2Api({ clientSecret })
+        const { access_token, expires_at: expiresAt, refresh_token } = await oauth2.refreshToken(currentRefreshToken)
+
+        await sbbolSecretStorage.setAccessToken(access_token, userId, { expiresAt })
+        await sbbolSecretStorage.setRefreshToken(refresh_token, userId)
+    }
+
+    return [await sbbolSecretStorage.getAccessToken(userId), await sbbolSecretStorage.getRefreshToken(userId)]
+}
+
+const bootstrap = async () => {
+    // get all keys by prefix
+    const [clientId] = process.argv.slice(3)
+
+    const allRefreshTokensKeys = await redisDB.keys(`SBBOL:auth:${clientId}:user:*refreshToken`)
+
+    const OAuthApi = await initializeSbbolAuthApi()
```

Comment:
Don't forget about the second integration
```suggestion
    const OAuthApi = await initializeSbbolAuthApi(useExtendedConfig)
```